src/pages/Quiz/index.css

```diff
@@ -0,0 +1,16 @@
+body,
+html,
+#root {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.quizGames {
+  margin: 0 auto;
+  max-width: 950px;
+}
```

Comment:
what do you think about unifying all of your styles?
instead of writing the same rules for all pages, you can write a root CSS file which includes all needed parameters (fe: `body`, `html`, etc.)

Reply:
Good idea I do this, Update in next commit